src/plugins/eqht-css/_base.scss

```diff
@@ -0,0 +1,25 @@
+.wb-eqht-grd{
+	align-items: stretch;
+	display: flex;
+	flex-wrap: wrap;
+	height: 100%;
+
+	>[class*="col-"]{
+		display:flex;
+		flex-direction: column;
+		flex-grow:1;
+	}
+}
+.wb-eqht-grd .hght-inhrt{
+	display: flex;
+	flex: 1 1 auto;
+	flex-direction: column;
+
+	> *{
+		display: flex;
+		flex-direction: column;
+	}
+}
+.wb-eqht-grd .hght-inhrt .eqht-trgt{
```

Comment:
This will need to be updated/removed because this is going to be completed by the plugin you created

Reply:
I don't understand.  The plugin only add the CSS class name "hght-inhrt" to facilitate implementation.  What needs to be removed/updated?

Reply:
OK.  It looks redundant here since flex: 1 1 auto is already in the parent .wb-eqht-grd .hght-inhrt but it doesn't work once removed...  Not sure what is overridding it.

No change made here

Reply:
The CSS class `.eqht-trgt` is only use for by the plugin. The `.hght-inhrt` CSS class do the same.